cron.yaml

```diff
@@ -0,0 +1,4 @@
+cron:
+# - description: "Watch every 1 minute"
+#   url: /watches/refresh
```

Comment:
@nagachika Yes, it is. I'm going to activate these lines after some tests.
Actually I'd like to set cron settings on deploy. Is it possible?